sensor_control/navigator_lidar_oa/src/astar_nav.cpp

```diff
@@ -0,0 +1,201 @@
+AStar astar(ROI_SIZE_METERS / VOXEL_SIZE_METERS);
```

Comment:
It was to separate functionality from lidar. 

However, Ira got rid of it, so just pretend it is not here.